erts/emulator/beam/erl_proc_sig_queue.h

```diff
@@ -927,6 +930,79 @@ erts_proc_sig_send_rpc_request(Process *c_p,
                                int reply,
                                Eterm (*func)(Process *, void *, int *, ErlHeapFragment **),
                                void *arg);
+/**
+ *
+ * @brief Send an 'rpc' signal to a process.
```

Comment:
Not that it matters a lot, but why break these comments at 50-60 characters? Even terminals aren't usually that small :)

Reply:
I usually try to break at 80, don't know why I didn't here when introducing the original comments for the other functions. This new comment just follows that style.